### ISSUE TITLE
[APP SUBMITTED]: AttributeError: 'list' object has no attribute 'season'

### ISSUE THREAD

**Issue by fernandog** [pymedusa/Medusa#1033]
### INFO

**Python Version**: `2.7.9 (default, Mar  1 2015, 13:48:22) [GCC 4.9.2]`
**Operating System**: `Linux-4.4.16-5-osmc-armv7l-with-debian-8.5`
**Locale**: `UTF-8`
**Branch**: [optimized](../tree/optimized)
**Commit**: PyMedusa/SickRage@89448604da63b78a49b5b93299cadf8ff406e473
**Link to Log**: https://gist.github.com/a8de6c277dfd6da0806a3692c6bcb817
### ERROR

<pre>
2016-09-05 11:46:47 ERROR    Thread-15 :: [8944860] Failed doing web ui callback: Traceback (most recent call last):
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/server/web/core/base.py#L270">sickbeard/server/web/core/base.py</a>", line 270, in async_call
    result = function(**kwargs)
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/server/web/home/handler.py#L1916">sickbeard/server/web/home/handler.py</a>", line 1916, in getManualSearchStatus
    episodes = collectEpisodesFromSearchThread(show)
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/manual_search.py#L159">sickbeard/manual_search.py</a>", line 159, in collectEpisodesFromSearchThread
    episodes += getEpisodes(search_thread, searchstatus)
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/manual_search.py#L98">sickbeard/manual_search.py</a>", line 98, in getEpisodes
    ep = show_obj.get_episode(ep_obj.season, ep_obj.episode)
AttributeError: 'list' object has no attribute 'season'
Traceback (most recent call last):
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/server/web/core/base.py#L270">sickbeard/server/web/core/base.py</a>", line 270, in async_call
    result = function(**kwargs)
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/server/web/home/handler.py#L1916">sickbeard/server/web/home/handler.py</a>", line 1916, in getManualSearchStatus
    episodes = collectEpisodesFromSearchThread(show)
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/manual_search.py#L159">sickbeard/manual_search.py</a>", line 159, in collectEpisodesFromSearchThread
    episodes += getEpisodes(search_thread, searchstatus)
  File "/home/osmc/SickRage/<a href="../blob/89448604da63b78a49b5b93299cadf8ff406e473/sickbeard/manual_search.py#L98">sickbeard/manual_search.py</a>", line 98, in getEpisodes
    ep = show_obj.get_episode(ep_obj.season, ep_obj.episode)
AttributeError: 'list' object has no attribute 'season'
</pre>

---

_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by ratoaq2** [comment]
I really liked the links in the new issue submitter :+1

**Comment by fernandog** [comment]
yes. but the files link wont work because branch `optimized` its not in pymedusa

**Comment by medariox** [comment]
@fernandog 
Maybe you need to use official Medusa builds 😜

**Comment by fernandog** [comment]
### INFO
**Python Version**: `2.7.9 (default, Aug 13 2016, 17:56:53) [GCC 4.9.2]`
**Operating System**: `Linux-4.4.16-5-osmc-armv7l-with-debian-8.6`
**Locale**: `UTF-8`
**Branch**: [optimized](../tree/optimized)
**Database**: `44.3`
**Commit**: pymedusa/SickRage@920c3a03cc5dd5213b3a514ea4816d2cbf937e1c
**Link to Log**: https://gist.github.com/6409d87a266b06741d9bbf9fcdbb9b5c
### ERROR
<pre>
2016-10-02 19:15:22 ERROR    FINDSUBTITLES :: [920c3a0] Traceback (most recent call last):
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/medusa/subtitles.py#L296">medusa/subtitles.py</a>", line 296, in download_subtitles
    for index, s in enumerate(subtitles_list)], key=operator.itemgetter(1), reverse=True)
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/lib/subliminal/score.py#L106">lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/lib/subliminal/providers/opensubtitles.py#L82">lib/subliminal/providers/opensubtitles.py</a>", line 82, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/lib/subliminal/subtitle.py#L237">lib/subliminal/subtitle.py</a>", line 237, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
Traceback (most recent call last):
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/medusa/subtitles.py#L296">medusa/subtitles.py</a>", line 296, in download_subtitles
    for index, s in enumerate(subtitles_list)], key=operator.itemgetter(1), reverse=True)
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/lib/subliminal/score.py#L106">lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/lib/subliminal/providers/opensubtitles.py#L82">lib/subliminal/providers/opensubtitles.py</a>", line 82, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "/home/osmc/SickRage/<a href="../blob/920c3a03cc5dd5213b3a514ea4816d2cbf937e1c/lib/subliminal/subtitle.py#L237">lib/subliminal/subtitle.py</a>", line 237, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by fernandog** [comment]
@ratoaq2 ^

**Comment by fernandog** [comment]
I added a log in subliminal and found out the culrpit:

```
Searching subtitles [{'moviebytesize': '725722727', 'sublanguageid': u'pob', 'moviehash': '15ec3ccdb0e43e31'}, {'sublanguageid': u'pob', 'tag': 'victoria.s01e07.720p.hdtv.x264-river.mkv'}, {'query': u'Victoria', 'sublanguageid': u'pob', 'episode': 7, 'season': 1}]
Found subtitle <OpenSubtitlesSubtitle '1954384994' [pt-BR]> by fulltext. Filename: Gossip Girl.S01E07.Victor, Victoria.WEB-DL 720p.br.srt
```

It returned a subtitle completly different from the search:
Gossip Girl.S01E07.Victor, Victoria.WEB-DL 720p.br.srt

```
python report_guessit.py "Gossip Girl.S01E07.Victor, Victoria.WEB-DL 720p.br.srt"
# guessit: 2.1.1.dev0  rebulk: 0.7.6
? Gossip Girl.S01E07.Victor, Victoria.WEB-DL 720p.br.srt
: title: Gossip Girl
  season: 1
  episode: 7
  episode_title: Victor, Victoria
  format: ['WEB-DL', 'BluRay']
  screen_size: 720p
  subtitle_language: pt-BR
  container: srt
  mimetype: text/plain
  type: episode
```

**Comment by fernandog** [comment]
i can't reproduce in subliminal CLI. Only in medusa

**Comment by fernandog** [comment]
Possible fix in subliminal/subtitle.py:
```diff
     # format
-    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
+    if video.format and 'format' in guess and not isinstance(guess['format'], list) and guess['format'].lower() == video.format.lower():

**Comment by NicoLeOca** [comment]
### INFO
**Python Version**: `2.7.11 (default, Dec 17 2015, 14:08:17) [GCC 4.6.3]`
**Operating System**: `Linux-3.2.40-i686-with-glibc2.0`
**Locale**: `UTF-8`
**Branch**: [develop](../tree/develop)
**Database**: `44.2`
**Commit**: pymedusa/SickRage@e7f252dca4a329b340cba51c5420feb18f3697b8
**Link to Log**: https://gist.github.com/0942c76e676a19f207bd2cde4df69d6c
### ERROR
<pre>
2016-10-05 17:27:09 ERROR    FINDSUBTITLES :: [e7f252d] Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/medusa/subtitles.py#L285">SickBeard/medusa/subtitles.py</a>", line 285, in download_subtitles
    for s in subtitles_list], key=operator.itemgetter(1), reverse=True)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/lib/subliminal/score.py#L106">SickBeard/lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/lib/subliminal/providers/opensubtitles.py#L78">SickBeard/lib/subliminal/providers/opensubtitles.py</a>", line 78, in get_matches
    matches |= guess_matches(video, guessit(self.movie_release_name, {'type': 'episode'}))
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/lib/subliminal/subtitle.py#L235">SickBeard/lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/medusa/subtitles.py#L285">SickBeard/medusa/subtitles.py</a>", line 285, in download_subtitles
    for s in subtitles_list], key=operator.itemgetter(1), reverse=True)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/lib/subliminal/score.py#L106">SickBeard/lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/lib/subliminal/providers/opensubtitles.py#L78">SickBeard/lib/subliminal/providers/opensubtitles.py</a>", line 78, in get_matches
    matches |= guess_matches(video, guessit(self.movie_release_name, {'type': 'episode'}))
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/e7f252dca4a329b340cba51c5420feb18f3697b8/SickBeard/lib/subliminal/subtitle.py#L235">SickBeard/lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by ratoaq2** [comment]
Subliminal didn't log the provider and the subtitle that caused that issue :-/
@NicoLeOca  If you could tell us with subtitle providers you have enabled and which languages, that's a start.

**Comment by NicoLeOca** [comment]
@ratoaq2 

![image](https://cloud.githubusercontent.com/assets/9487685/19222422/42f6200a-8e58-11e6-8eaf-c7e93f47212b.png)

english subs only

**Comment by ratoaq2** [comment]
Thanks, I found the guilty one in opensubtitles. A subtitle with 2 formats: HDTV and WEB-DL

`Preacher.S01E10.HDTV.x264.WEB-DL.HI.C`

**Comment by ratoaq2** [comment]
That was fixed here: https://github.com/pymedusa/SickRage/pull/1234/commits/627d7a76ec9742ef4e87d0e0d5f71dfa9532a86a

It will soon reach develop branch

**Comment by NicoLeOca** [comment]
@ratoaq2 
thanks and well done

**Comment by NicoLeOca** [comment]
### INFO
**Python Version**: `2.7.11 (default, Dec 17 2015, 14:08:17) [GCC 4.6.3]`
**Operating System**: `Linux-3.2.40-i686-with-glibc2.0`
**Locale**: `UTF-8`
**Branch**: [develop](../tree/develop)
**Database**: `44.2`
**Commit**: pymedusa/Medusa@f950560c7aac7c89cbbbd716bb5809ac5fca4e4a
**Link to Log**: https://gist.github.com/2171772084a48d434a8116346daa9fc0
### ERROR
<pre>
2016-10-16 11:33:18 ERROR    FINDSUBTITLES :: [f950560] Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/medusa/subtitles.py#L375">SickBeard/medusa/subtitles.py</a>", line 375, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/medusa/subtitles.py#L244">SickBeard/medusa/subtitles.py</a>", line 244, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/lib/subliminal/score.py#L106">SickBeard/lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/lib/subliminal/providers/opensubtitles.py#L79">SickBeard/lib/subliminal/providers/opensubtitles.py</a>", line 79, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/lib/subliminal/subtitle.py#L235">SickBeard/lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/medusa/subtitles.py#L375">SickBeard/medusa/subtitles.py</a>", line 375, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/medusa/subtitles.py#L244">SickBeard/medusa/subtitles.py</a>", line 244, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/lib/subliminal/score.py#L106">SickBeard/lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/lib/subliminal/providers/opensubtitles.py#L79">SickBeard/lib/subliminal/providers/opensubtitles.py</a>", line 79, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "/volume1/@appstore/sickbeard-custom/var/<a href="../blob/f950560c7aac7c89cbbbd716bb5809ac5fca4e4a/SickBeard/lib/subliminal/subtitle.py#L235">SickBeard/lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by fernandog** [comment]
@ratoaq2 still need to fix this one ? ^

**Comment by NicoLeOca** [comment]
### INFO
**Python Version**: `2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:24:40) [MSC v.1500 64 bit (AMD64)]`
**Operating System**: `Windows-10-10.0.14393`
**Locale**: `cp1252`
**Branch**: [develop](../tree/develop)
**Database**: `44.2`
**Commit**: pymedusa/Medusa@3d784e1e77e78bf389935e0dc3c2ab27772eed78
**Link to Log**: https://gist.github.com/404919324fd5f06b256c212e8ee10210
### ERROR
<pre>
2016-10-23 05:37:56 ERROR    FINDSUBTITLES :: [3d784e1] Traceback (most recent call last):
  File "C:\Medusa\medusa\subtitles.py", line 375, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "C:\Medusa\medusa\subtitles.py", line 244, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "C:\Medusa\lib\subliminal\score.py", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "C:\Medusa\lib\subliminal\providers\opensubtitles.py", line 79, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "C:\Medusa\lib\subliminal\subtitle.py", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
Traceback (most recent call last):
  File "C:\Medusa\medusa\subtitles.py", line 375, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "C:\Medusa\medusa\subtitles.py", line 244, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "C:\Medusa\lib\subliminal\score.py", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "C:\Medusa\lib\subliminal\providers\opensubtitles.py", line 79, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "C:\Medusa\lib\subliminal\subtitle.py", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by duramato** [comment]
### INFO
**Python Version**: `2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]`
**Operating System**: `Linux-4.2.0-42-generic-i686-with-Ubuntu-14.04-trusty`
**Locale**: `None`
**Branch**: [develop](../tree/develop)
**Database**: `44.2`
**Commit**: pymedusa/Medusa@3399717a048c7e9e95648aa5af99f1f864891738
**Link to Log**: https://gist.github.com/94f30954d0f13871aee041425d95c393
### ERROR
<pre>
2016-10-24 20:12:18 ERROR    FINDSUBTITLES :: [3399717] Traceback (most recent call last):
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/medusa/subtitles.py#L375">medusa/subtitles.py</a>", line 375, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/medusa/subtitles.py#L244">medusa/subtitles.py</a>", line 244, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/lib/subliminal/score.py#L106">lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/lib/subliminal/providers/legendastv.py#L147">lib/subliminal/providers/legendastv.py</a>", line 147, in get_matches
    matches |= guess_matches(video, guessit(self.archive.name, {'type': self.type}))
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/lib/subliminal/subtitle.py#L235">lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
Traceback (most recent call last):
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/medusa/subtitles.py#L375">medusa/subtitles.py</a>", line 375, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/medusa/subtitles.py#L244">medusa/subtitles.py</a>", line 244, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/lib/subliminal/score.py#L106">lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/lib/subliminal/providers/legendastv.py#L147">lib/subliminal/providers/legendastv.py</a>", line 147, in get_matches
    matches |= guess_matches(video, guessit(self.archive.name, {'type': self.type}))
  File "/home/home/SickRage/<a href="../blob/3399717a048c7e9e95648aa5af99f1f864891738/lib/subliminal/subtitle.py#L235">lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by NicoLeOca** [comment]
### INFO
**Python Version**: `2.7.12 (default, Jul  1 2016, 15:12:24) [GCC 5.4.0 20160609]`
**Operating System**: `Linux-4.4.0-47-generic-x86_64-with-Ubuntu-16.04-xenial`
**Locale**: `UTF-8`
**Branch**: [develop](../tree/develop)
**Database**: `44.2`
**Commit**: pymedusa/Medusa@547bb6076a33d8dd526f48d13a7f1750c7c76b4a
**Link to Log**: https://gist.github.com/16d7e12f96da7f1af934d45f1c904d83
### ERROR
<pre>
2016-11-14 11:01:08 ERROR    FINDSUBTITLES :: [547bb60] Traceback (most recent call last):
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/medusa/subtitles.py#L426">medusa/subtitles.py</a>", line 426, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/medusa/subtitles.py#L295">medusa/subtitles.py</a>", line 295, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/lib/subliminal/score.py#L106">lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/lib/subliminal/providers/opensubtitles.py#L79">lib/subliminal/providers/opensubtitles.py</a>", line 79, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/lib/subliminal/subtitle.py#L235">lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
Traceback (most recent call last):
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/medusa/subtitles.py#L426">medusa/subtitles.py</a>", line 426, in download_subtitles
    scored_subtitles = score_subtitles(subtitles_list, video)
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/medusa/subtitles.py#L295">medusa/subtitles.py</a>", line 295, in score_subtitles
    for s in subtitles_list], key=operator.itemgetter(1, 2), reverse=True)
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/lib/subliminal/score.py#L106">lib/subliminal/score.py</a>", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/lib/subliminal/providers/opensubtitles.py#L79">lib/subliminal/providers/opensubtitles.py</a>", line 79, in get_matches
    matches |= guess_matches(video, guessit(self.filename, {'type': 'episode'}))
  File "/opt/Medusa/<a href="../blob/547bb6076a33d8dd526f48d13a7f1750c7c76b4a/lib/subliminal/subtitle.py#L235">lib/subliminal/subtitle.py</a>", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by ratoaq2** [comment]
This one is related to opensubtitle release names. Some of them ends up with multiple formats and subliminal doesn't support that. 

What I can do is to create a rule and drop the formats and log a warning, or error. Probably an error, so users can submit them and I can keep track of the problematic  scenarios

**Comment by ratoaq2** [comment]
Should be fixed by #1544
